src/UI/components/board.ts

```diff
@@ -17,6 +16,7 @@ function BoardContainer(props: BoardContainerProps) {
     let mouseYIndex = false;
     let mouseTimeout: number;
     let mouseStatus = 'up';
+    let cellSize = 50;
```

Comment:
Is 50 the best default size? It's pretty big, maybe we should go down a little...